### PR TITLE
VUE-510 Update Husky and fix linting hook

### PR DESCRIPTION
**Pull request by eddieferrer**
* Fixes CheckoutDropIn formatting
* Update husky to v5
* fix husky hooks

### CODE REVIEW

File: src/components/Checkout/CheckoutDropInPaymentWrapper.vue [hunk 1]
```diff
@@ -27,23 +27,32 @@
 					name="termsAgreement"
```
Comment (eddieferrer) [inline src/components/Checkout/CheckoutDropInPaymentWrapper.vue:27]: Only auto formatting changes for this file. Not sure why the formatting was so whacky.

**Comment by eddieferrer** [comment]
`npm run lint` is a tad slower than I would like. We could alternatively only lint staged files on commit instead of pre-push. This would lint significantly faster, but it would not allow committing files with lint errors, which could be an inconvenience with work in progress commits.

**Review by ryan-ludwig — APPROVED** [review]

**Review by BoulderBrains — APPROVED** [review]

**Review by mcstover — APPROVED** [review]
We can test out the time difference for a little while then look into tuning the linting a bit.

**Review by emuvente — COMMENTED** [review]
👍

**Pull request merged**